7024Movies/Pages/MovieReviewsComedy/Index.cshtml

```diff
@@ -31,11 +31,9 @@
             <td>
                 @Html.DisplayFor(modelItem => item.MovieReviewC)
             </td>
-            <td>
-                <a asp-page="./Pages/DiscoverThriller">Return to Comedy Ratings</a> |
-
-            </td>
         </tr>
 }
     </tbody>
 </table>
+
+<a class="btn btn-dark" href="/DiscoverComedy" role="button">Return to Comedy Ratings</a>
```

Comment:
I added this button link to DiscoverComedy and removed the regular text link above for easier access. I also added in the correct reference to the page since they had it going to another page